components/members/upload/forms/CountableTextArea.tsx

```diff
@@ -1,31 +1,32 @@
 import { css } from '@emotion/react';
 import styled from '@emotion/styled';
-import { forwardRef, TextareaHTMLAttributes, useState } from 'react';
+import { forwardRef } from 'react';
 
 import Text from '@/components/common/Text';
 import { colors } from '@/styles/colors';
 import { MOBILE_MEDIA_QUERY } from '@/styles/mediaQuery';
 import { textStyles } from '@/styles/typography';
 
-interface MemberCountableTextAreaProps extends TextareaHTMLAttributes<HTMLTextAreaElement> {
+interface MemberCountableTextAreaProps {
   error?: boolean;
   maxCount?: number;
+  onChange: () => void;
```

Comment:
p2; 정확한 타입을 위해 이렇게 바꿔주면 좋을 것 같아요~

```suggestion
  onChange: (e: ChangeEvent) => void;
```

Reply:
오오 좋아용